### ISSUE TITLE
add mono to builds

### ISSUE THREAD

**Issue by grusin** [jens-maus/RaspberryMatic#546]
I would like to request adding 'mono' package to the default build. I am using home automation code that i have written in .net 4.7.2 and it runs perfectly fine both on windows and linux (incl. rpi3) machines.

Currently I am making my own images based on official releases, with just ticking the 'mono' checkbox in the buildroot.

If that's not a problem, could you please include it in your default build?

Thanks

**Comment by jens-maus** [comment]
Nope, sorry that won't happen because we also don't have python, perl, etc. and have to maintain compatibility to the CCU3 firmware as much as possible. So sorry, but this won't be added as a default package since there is no internal dependency requiring mono or C#. So please create an own CCU Mono Addon package if you want to supply this to other users so that mono will be installed in /usr/local, etc.
